python/python/tests/test_async_client.py

```diff
@@ -1259,6 +1259,31 @@ async def test_echo(self, redis_client: TRedisClient):
         message = get_random_string(5)
         assert await redis_client.echo(message) == message
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_dbsize(self, redis_client: TRedisClient):
+        assert await redis_client.custom_command(["FLUSHALL"]) == OK
+
+        key_value_pairs = [(get_random_string(10), "foo") for _ in range(10)]
+        assert await redis_client.dbsize() == 0
+
+        for key, value in key_value_pairs:
+            assert await redis_client.set(key, value) == OK
+        assert await redis_client.dbsize() == 10
+
+        if isinstance(redis_client, RedisClusterClient):
```

Comment:
Tests are so different for cluster and standalone clients, maybe split them into two test cases?